src/com/company/Metods.java

```diff
@@ -0,0 +1,215 @@
+package com.company;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public  class Metods {
+    public static void PrintBoard(String X[][]) {
```

Comment:
В Java методы именуются с применением CamelCase, но первое слово - с маленькой буквы - `printBoard`

```suggestion
    public static void printBoard(String X[][]) {
```